bids/modeling/auto_model.py

```diff
@@ -3,9 +3,15 @@
 import numpy as np
 
 
-def _make_passthrough_contrast(level, contrast_names, test="t"):
-    block = OrderedDict(Level=level, Name=level,
-                        Model={'X': contrast_names})
+def _make_passthrough_contrast(level, contrast_names, model_type='meta', test="t"):
+    gb_dict = dict(Subject=['subject', 'contrast'],
+                   Session=['session', 'contrast'],
+                   Dataset=['contrast'])
+    block = OrderedDict(Level=level,
+                        Name=level,
+                        GroupBy=gb_dict[level],
+                        Model={'Type': model_type,
+                               'X': contrast_names})
```

Comment:
Dicts are ordered. Let's get rid of them when we need to touch them.

```suggestion
    block = dict(Level=level,
                 Name=level,
                 GroupBy=gb_dict[level],
                 Model={'Type': model_type,
                        'X': contrast_names})
```